src/main/resources/db/migration/V0_7__Update_fee_table.sql

```diff
@@ -0,0 +1,18 @@
+alter table "fee" add column if not exists remaining_amount integer;
+alter table "fee" add column if not exists status_updatedtime timestamp without time zone default now();
+
+create or replace function update_status_updatedtime()
+    returns trigger as
+$$
+begin
+    new.status_updatedtime = now();
+    return new;
+end;
+$$ language 'plpgsql';
+
+create trigger update_status_updated_time
+    before update
+    on
+        "fee"
+    for each row
+execute procedure update_status_updatedtime();
```

Comment:
So you mean, if I update the comment of the fee, the `status_updated_time` is updated ? 

Nor you should rename the status_updated_time to `updatedAt`

Nor you should not update the status_updated_time for EVERY update, only when update is set on fee_status.